shopfloor/actions/message.py

```diff
@@ -481,16 +475,10 @@ def lot_multiple_packages_scan_package(self):
             "body": _("This lot is part of multiple packages, please scan a package."),
         }
 
-    def lot_not_found(self):
```

Comment:
removing methods w/o deprecation might break existing implementations

Reply:
True.  I checked and it is not being used in this module or any merged related modules. I thought doing good by cleaning up.
What is the the proper way to do this, then ? Knowing that this module is still in it beta https://github.com/OCA/wms/blob/0c854c54458c932ea6ecdfa4d6e0f0bd5c5673ab/shopfloor/__manifest__.py#L10

Reply:
Yeah, we should change this to stable indeed :sweat_smile: 
However, in general, even if it's beta I tend to preserve backward compat when it's easy.
Now, this is a small thing and if you checked is not used anywhere... that's ok.
Let's move on :)